node/src/store.rs

```diff
@@ -227,8 +268,8 @@ pub mod tests {
     #[test]
     fn test_empty_store() {
         let s = InMemoryStore::new();
-        assert_eq!(s.get_head_height(), 0);
-        assert!(matches!(s.get_head(), Err(StoreError::NotFound)));
+        assert!(matches!(s.get_head_height(), Err(StoreError::EmptyStore)));
+        assert!(matches!(s.get_head(), Err(StoreError::EmptyStore)));
         assert!(matches!(s.get_by_height(1), Err(StoreError::NotFound)));
```

Comment:
If we have separate store error for `EmptyStore`, would it make sense for it to be returned for all operations and let the caller decide whether to handle that differently?

Reply:
I think if I was a user I wouldn't like to get `EmptyStore` when I do `get_by_height`. We can remove `EmptyStore` and  `NotFound` only, but we should document this in `get_head`